decode_symphonia/src/lib.rs

```diff
@@ -341,23 +341,24 @@ impl Drop for SymphoniaDecoder {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use float_cmp::*;
 
     #[test]
     fn decoder_new() {
         let files = vec![
             //  file | num_channels | num_frames | sample_rate
-            ("./test_files/wav_u8_mono.wav", 1, 1323000, Some(44100)),
-            ("./test_files/wav_i16_mono.wav", 1, 1323000, Some(44100)),
-            ("./test_files/wav_i24_mono.wav", 1, 1323000, Some(44100)),
-            ("./test_files/wav_i32_mono.wav", 1, 1323000, Some(44100)),
-            ("./test_files/wav_f32_mono.wav", 1, 1323000, Some(44100)),
-            ("./test_files/wav_i24_stereo.wav", 2, 1323000, Some(44100)),
-            //"./test_files/ogg_mono.ogg",
-            //"./test_files/ogg_stereo.ogg",
-            //"./test_files/mp3_constant_mono.mp3",
-            //"./test_files/mp3_constant_stereo.mp3",
-            //"./test_files/mp3_variable_mono.mp3",
-            //"./test_files/mp3_variable_stereo.mp3",
+            ("../test_files/wav_u8_mono.wav", 1, 1323000, Some(44100)),
```

Comment:
Is using relative paths robust? Does this fail when running `cargo test` anywhere besides the root of the repository? I think it might be better to use a path relative to the `CARGO_MANIFEST_DIR` environment variable.

Reply:
For me it works in both the decode_symphonia folder and the root folder.